book/accessibility.md

```diff
@@ -2351,6 +2337,18 @@ instead. Implement it.
 
 [hover-pseudo]: https://developer.mozilla.org/en-US/docs/Web/CSS/:hover
 
+* *Alert updates*: Right now, the screen-reader immediately reads an
+alert when an element gains the `alert` role. But if that element's
+contents change, the alert isn't re-read. In real browsers, changes to
+an alert cause the alert to be re-read. Implement that in your
+browser: if the text description of any node below an `alert` node
+changes in the accessibility tree, re-read the `alert`. For an
```

Comment:
I don't think our runtime currently supports changing out the text without the element...also add that API?

Reply:
You can change the `innerHTML` of an existing element to regular text. It won't change the element itself, just delete existing children and add a new one.

Reply:
Right, forgot about that.